src/CMakeLists.txt

```diff
@@ -182,7 +185,7 @@ if(${CMAKE_CXX_COMPILER_ID} STREQUAL "MSVC")
   endif()
 else()
   # clang/gcc
-  set(COMMON_COMPILE_FLAGS "-fPIC -std=c++11 -m${BITNESS} ${STDLIB} -fvisibility=hidden -Werror -Wall -Wextra -Wreturn-type -Wunused -Wno-unused-parameter -Wno-missing-field-initializers")
+  set(COMMON_COMPILE_FLAGS "-fPIC -std=c++11 -m${BITNESS} ${STDLIB} -fvisibility=hidden -Wall -Wextra -Wreturn-type -Wunused -Wno-unused-parameter -Wno-missing-field-initializers")
```

Comment:
Can we keep `-Werror` in for the core C++ build and just remove it for the SWIG build?